src/tseda/datastore.py

```diff
@@ -71,31 +70,31 @@ class IndividualsTable(Viewer):
         default=100,
```

Comment:
should the default maybe be less than 100? It's set to 20 on the sample sets page

Reply:
True, though this dataset is very small, its probably nicer to not load too many samples at once